src-self-hosted/main.zig

```diff
@@ -525,6 +525,7 @@ const usage_fmt =
 
 const args_fmt_spec = []Flag{
     Flag.Bool("--help"),
+    Flag.Bool("--check-formatted"),
```

Comment:
Let's go with simply `--check`. This flag is namespaced under `zig fmt` so the concept of "format" is already implied